src/ast/builder.rs

```diff
@@ -67,6 +67,11 @@ impl CompleteAtom {
     pub fn or(self) -> IncompleteOr<Builder, Self> {
         IncompleteOr::new(Builder, self)
     }
+
+    /// Append an `Implication` to the sentence.
```

Comment:
In these docstrings I'm referencing the actual type names.

```suggestion
    /// Append an `Implies` to the sentence.
```

Same for the ones below.

Reply:
I might end up changing all of these to actual links to the types. 🤔